README.md

```diff
@@ -25,7 +25,7 @@ npm install @guidesmiths/license-checker
 - Add a script to run the package
 
 ```
-npx @guidesmiths/license-checker --failOn license1,license2
+npx @guidesmiths/license-checker --failOn license1 license2
```

Comment:
this would be a breaking change right?

Reply:
Yep, we'll need to release v2.0.0

Reply:
Sure. How do you propose to do it? In the same README, create a new document for migration guides, or others?

Reply:
Perhaps a new doc for migration guides would be a good idea. What do you think?